source/skyland/scene/fadeInScene.cpp

```diff
@@ -159,6 +160,10 @@ ScenePtr FadeInScene::update(Time delta)
             APP.time_stream().enable_pushes(false);
```

Comment:
I guess, now that the shop script disables rewind explicitly, we can remove this `else if` check for disabling rewind for shops